pydantic/main.py

```diff
@@ -853,6 +860,8 @@ def validate_model(  # noqa: C901 (ignore complexity)
                     errors.append(ErrorWrapper(ExtraError(), loc=f))
 
     for validator in model.__post_root_validators__:
+        if errors and getattr(validator, Validator.SKIP_ON_FAILURE_KEY, False):
```

Comment:
I think this can then become
```suggestion
        if errors and getattr(validator, Validator.ROOT_VALIDATOR_CONFIG_KEY).skip_on_failure:
```